server/dash/pages/nc_summary_comparison.py

```diff
@@ -58,6 +58,133 @@ def merge_dict(d1, d2):
 COMPARISON_DASHBOARD_TITLE = "LA 311 Requests - Neighborhood Council Comparison Dashboard"
 
 
+## DATA WRANGLING HELPER FUNCTIONS.
+def add_datetime_column(df, colname):
```

Comment:
Let's make this return a tuple of (df, newColname) so that the callers don't need to manually construct the newColname themselves.

Reply:
thanks for the suggestion - resolved.